Source/SpaceMouse/Public/SpaceMouseConfig.h

```diff
@@ -15,6 +15,7 @@
 
 class FInputBindingManager;
 class FUICommandInfo;
+enum class EV3DCmd : uint8;
```

Comment:
forward declaring enums are not the most reliable thing to do. just do
```
#include "SmButtons.h"
```
before the generated header

Reply:
I do notthink there is a SmButtons.h in this release. Perhaps Buttons.h

Reply:
The sources compile with this header on my system